controllers/configurationpolicy_controller.go

```diff
@@ -2123,10 +2114,10 @@ func (r *ConfigurationPolicyReconciler) createParentPolicyEvent(instance *policy
 		eventMsg)
 }
 
-func createParentPolicy(instance *policyv1.ConfigurationPolicy) extpoliciesv1.Policy {
+func createParentPolicy(instance *policyv1.ConfigurationPolicy, defaultNamespace string) extpoliciesv1.Policy {
```

Comment:
Nit: can we add doc for when/why the defaultNamespace is used?

Reply:
I'm going to refactor this so that we just use the namespace from the `ConfigurationPolicy` object since that's always going to be the watch namespace anyways.